api.md

```diff
@@ -1685,53 +1686,41 @@ GROUP BY <time_bucket( <const_value>, <partition_col_of_hypertable> ),
-|   |   |   |
-|**Name**|||
 |`timescaledb.create_group_indexes`|||
 |**Description**|**Type**|**Default**|
 | Create indexes on the materialization table for the group by columns (specified by the `GROUP BY` clause of the `SELECT` query). | `BOOLEAN` | Indexes are created by default for every group by expression + time_bucket expression pair.|
-|   |   |   |
-|**Name**|||
-|`timescaledb.ignore_invalidation_older_than`|||
-|**Description**|**Type**|**Default**|
-| Time interval after which invalidations are ignored.| Same datatype as the `bucket_width` argument from the `time_bucket` expression. | By default all invalidations are processed.|
 
->:TIP: Say, the continuous aggregate uses time_bucket(INTERVAL '2h', time_column) and we want to keep the view up to date with the data. We can do this by modifying the `refresh_lag` setting. Set refresh_lag to `-2h`. E.g. `ALTER VIEW contview set (timescaledb.refresh_lag = '-2h');` Please refer to the [caveats][].
+#### Notes
 
-#### Restrictions
+- The view will be automatically refreshed (as outlined under
+  [`refresh_continuous_aggregate`](#refresh_continuous_aggregate))
+  unless `WITH NO DATA` is given (`WITH DATA` is the default).
 - `SELECT` query should be of the form specified in the syntax above.
-- The hypertable used in the `SELECT` may not have [row-level-security policies][postgres-rls] enabled.
--  `GROUP BY` clause must include a time_bucket expression. The [`time_bucket`](#time_bucket) expression must use the time dimension column of the hypertable.
-- [`time_bucket_gapfill`](#time_bucket_gapfill) is not allowed in continuous
-  aggs, but may be run in a `SELECT` from the continuous aggregate view.
-- In general, aggregates which can be [parallelized by PostgreSQL][postgres-parallel-agg] are allowed in the view definition, this
-  includes most aggregates distributed with PostgreSQL. Aggregates with `ORDER BY`,
-  `DISTINCT` and `FILTER` clauses are not permitted.
-* All functions and their arguments included in `SELECT`, `GROUP BY` and `HAVING` clauses must be [immutable][postgres-immutable].
+- The hypertable used in the `SELECT` may not have [row-level-security
+  policies][postgres-rls] enabled.
+-  `GROUP BY` clause must include a time_bucket expression. The
+   [`time_bucket`](#time_bucket) expression must use the time
+   dimension column of the hypertable.
+- [`time_bucket_gapfill`](#time_bucket_gapfill) is not allowed in
+  continuous aggs, but may be executed by a `SELECT` on the continuous
+  aggregate view.
+- In general, aggregates which can be [parallelized by
+  PostgreSQL][postgres-parallel-agg] are allowed in the view
+  definition, this includes most aggregates distributed with
+  PostgreSQL. Aggregates with `ORDER BY`, `DISTINCT` and `FILTER`
+  clauses are not permitted.
+- All functions and their arguments included in `SELECT`, `GROUP BY`
+  and `HAVING` clauses must be [immutable][postgres-immutable].
 - Queries with `ORDER BY` are disallowed.
```

Comment:
Minor nit:  What's the difference between these line:

     Queries with `ORDER BY` are disallowed.

And the above:

    Aggregates with `ORDER BY`, `DISTINCT` and `FILTER` clauses are not permitted.

Reply:
Seems like it was accidentally duplicated. Removed the second one.